docs/schema_additions.md

```diff
@@ -0,0 +1,132 @@
+# Purpose
+
+This document describes the database specific additions to the
+[OSV Schema](https://github.com/ossf/osv-schema) specific to the Malicious
+Packages repository.
+
+The [OSV Schema](https://github.com/ossf/osv-schema) is a format specification
+for reporting vulnerabilities in Open Source projects. This format has been
+chosen to work with existing OSV tooling.
+
+The Malicious Packages repository uses the OSV format to store reports of
+malicious packages published on open source repositories.
+
+The additions described in this document have two primary functions:
+
+1. Annotating reports
+1. Managing the database
+
+To understand how the standard OSV fields are used, see the
+[Contributing Guide](https://github.com/ossf/malicious-packages/blob/main/CONTRIBUTING.md) for documentation.
+
+# Database Specific Format Overview
+
+The underlying OSV format is a JSON-based encoding format. Additions are
+expressed in the following informal schema. The exact details of each field are
+elaborated in the next section. All strings contain UTF-8 text.
+
+```json
+{
+    /* ... other OSV fields ... */
+    "database_specific": {
+	    "malicious-packages-origins" : [ {
+			"source": string,
+			"sha256": string,
+			"import_time": string,
+			"modified_time": string,
+			"ranges": [ /* OSV range object */ ],
+			"versions": [ string ]
+		} ],
+		"iocs": {
+			"domains": [ string ],
+			"urls": [ string ],
+			"ip_addresses": [ string ],
+		},
+		/* ... other database specific entries ... */
+    }
+}
+```
+
+# Field Details
+
+## malicious-packages-origins fields
+
+The `malicious-packages-origins` field is a JSON array containing objects that
```

Comment:
Nit: should we state up front that this is likely not useful for most consumers, and is more internal state for the malicious-packages repo? 

Also consider moving this to the last section as well, as otherwise readers might get lost in this before they get to the ones that are more interesting to them.

Reply:
I moved it lower and updated the first paragraph to indicate more clearly it is for internal use.